### DISCUSSION TITLE
Salad Fingers

### DISCUSSION THREAD

**Discussion by tjex** [tridactyl/tridactyl#5030] [General]
Just found the salad fingers shortcut when exploring `:viewconfig --user` binds hahaha.

What a weird surprise. That takes me waaaaay back! Couldn't not comment about this. Too good.

**Comment by bovine3dom** [comment]
If you're wondering why we included it, it's because salad fingers is a tridactyl :)

**Reply by tjex** [reply]
haha true! And so the nightmares begin of waking up in the middle of night to see salad fingers sitting at my desk, illuminated by the glow of the screen, delicately tapping away on my keyboard and staring straight at me ⚰️
